org/lateralgm/subframes/SpriteFrame.java

```diff
@@ -1269,8 +1268,11 @@ else if (cmd.endsWith(".PASTE"))
 					LGM.showDefaultExceptionHandler(e);
 					}
 				imageChanged = true;
-				res.subImages.addAll(pos + 1,images.bi);
-				subList.setSelectionInterval(pos + 1,pos + images.bi.size());
+				for (int i = 0; i < images.bi.size(); i++)
+					{
+					res.subImages.add(pos + i, Util.cloneImage(images.bi.get(i)));
+					}
+				subList.setSelectionInterval(pos, pos + images.bi.size() - 1);
```

Comment:
Why did this change?

Reply:
Say you have the first subimage selected when you hit paste, the subimage you are pasting should take its place, and the one that was there should be moved down one. Previously it would paste one after the current selection. So I basically just undid that. Don't forget, I'm the one who wrote the original code for this part here.